test/distributed/test_c10d.py

```diff
@@ -1936,7 +1936,7 @@ def forward(self, x):
         return self.p + x
 
 
-class TestDdpCommHook(nn.Module):
+class DdpComHook(nn.Module):
```

Comment:
com -> comm

Shall we rename this to `ModuleForDdpCommHook` as this is a module instead of a hook.